README.md

```diff
@@ -8,18 +8,18 @@ We build on the solders of giants with the following technologies:
 
 * [React](https://github.com/facebook/react)
 * [React Router](https://github.com/rackt/react-router)
-* [JSON Web Token](https://www.npmjs.com/package/jsonwebtoken) JSON Web Token for API authentication
 * [Babel](http://babeljs.io) for ES6 and ES7 magic
 * [Webpack](http://webpack.github.io) for bundling
 * [Webpack Dev Middleware](http://webpack.github.io/docs/webpack-dev-middleware.html)
+* [Clean Webpack Plugin](https://github.com/johnagan/clean-webpack-plugin)
 * [Redux](https://github.com/rackt/redux)'s futuristic [Flux](https://facebook.github.io/react/blog/2014/05/06/flux.html) implementation
 * [Redux Dev Tools](https://github.com/rackt/redux-devtools) for next generation DX (developer experience). Watch [Dan Abramov's talk](https://www.youtube.com/watch?v=xsSnOQynTHs)
 * [Redux Thunk](https://github.com/gaearon/redux-thunk) Thunk middleware for Redux - used in async actions
 * [React Router Redux](https://github.com/rackt/react-router-redux) Ruthlessly simple bindings to keep react-router and redux in sync
 * [fetch](https://github.com/github/fetch) A window.fetch JavaScript polyfill
 * [style-loader](https://github.com/webpack/style-loader), [sass-loader](https://github.com/jtangelder/sass-loader) and [less-loader](https://github.com/webpack/less-loader) to allow import of stylesheets in plain css, sass and less,
 * [font-awesome-webpack](https://github.com/gowravshekar/font-awesome-webpack) to customize FontAwesome
-* [foundation-sites](https://github.com/zurb/foundation-sites) for the grid
+* [bootstrap-sass-loader](https://github.com/shakacode/bootstrap-sass-loader) to customize Bootstrap
```

Comment:
I guess git is wrong, see deprecation message
https://github.com/shakacode/bootstrap-sass-loader